app/Integrations/Base/WebhookPlugin.php

```
@@ -67,6 +67,12 @@ public function handleWebhook(Request $request, Integration $integration): void
         $this->createEventsFromWebhook($convertedData, $integration);
     }
 
+    public function verifyWebhookSignature(Request $request, Integration $integration): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)